### PR TITLE
BD-1782: Update add-to-calendar-link additional options

### PR DESCRIPTION
**Pull request by KellieHawks**
# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [ ] Check that all links work.
- [ ] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @Timothy-Kim and @KellieHawks as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the internal reviewing chart to tag the appropriate reviewer.
- [ ] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Vercel environment by clicking the preview link in the vercel-bot comment in your PR.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @timothy-kim or @KellieHawks for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@josh-mccrowell-braze</td>
    <td>Monolith Deployments<br>Quality Infrastructure<br>Platform Infrastructure<br>Datalake<br>SDKs</td>
  </tr>
  <tr>
    <td>@kelliehawks</td>
    <td>Currents<br>Internal Tools<br>Product Partnerships<br>SMS<br>Customer Lifecycle, Identity and Permissions</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Reporting<br>Intelligence<br>User Targeting<br>IAM<br>Channels<br>FIX</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Ingestion<br>Core Objects<br>Core Messaging<br>Messaging and Automation<br>Email (Composition and Infrastructure)</td>
  </tr>
</table>
</details>

### CODE REVIEW

File: _docs/_user_guide/engagement_tools/campaigns/ideas_and_strategies/add_to_calendar_links.md [hunk 1]
```diff
@@ -55,8 +55,16 @@ https://ics.agical.io/?subject=Meet%20Braze&location=114%20Sansome%20Street&dtst
...
 ### Additional parameters
 
-You can add additional parameters for recurring events.
+The following parameters are optional and can be used to define additional aspects of an event.
 
+**Organizer name:** `&organizer=name`
+**Attach URL related to event:** `&attach=http://www.example.com/`
+**Duration:** `duration=30M`, as an alternative to dtend specify a duration like 1H or 30M
+**Reminder alarm time, in minutes:** `&reminder=15`
+**All day event:** `&allday=1` for all day event
```
Comment (KellieHawks) [inline _docs/_user_guide/engagement_tools/campaigns/ideas_and_strategies/add_to_calendar_links.md:64]:
```suggestion
**All day event:** `&allday=1`
```

File: _docs/_user_guide/engagement_tools/campaigns/ideas_and_strategies/add_to_calendar_links.md [hunk 2]
```diff
@@ -55,8 +55,16 @@ https://ics.agical.io/?subject=Meet%20Braze&location=114%20Sansome%20Street&dtst
 
 ### Additional parameters
 
-You can add additional parameters for recurring events.
+The following parameters are optional and can be used to define additional aspects of an event.
 
+**Organizer name:** `&organizer=name`
+**Attach URL related to event:** `&attach=http://www.example.com/`
+**Duration:** `duration=30M`, as an alternative to dtend specify a duration like 1H or 30M
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/engagement_tools/campaigns/ideas_and_strategies/add_to_calendar_links.md:62]: Just a small typo but I'm not sure what "dtend" is supposed to be

[the remark this answers is not in the document]
Comment (bre-fitzgerald) [reply _docs/_user_guide/engagement_tools/campaigns/ideas_and_strategies/add_to_calendar_links.md:62]: Oh!! Okay my mistake, maybe we should make it code text? I like what you put here explicitly stating it, thoughts on this? 

> ...as an alternative to the event end time (`dtend`), specify ....

[the remark this answers is not in the document]
Comment (KellieHawks) [reply _docs/_user_guide/engagement_tools/campaigns/ideas_and_strategies/add_to_calendar_links.md:62]: Good call!

Comment (KellieHawks) [inline _docs/_user_guide/engagement_tools/campaigns/ideas_and_strategies/add_to_calendar_links.md:62]:
```suggestion
**Duration:** `duration=30M`, as an alternative to the event end time (dtend), specify a duration like 1H or 30M
```

**Pull request merged**